kratos/utilities/coordinate_transformation_utilities.h

```diff
@@ -717,10 +707,12 @@ class CoordinateTransformationUtils {
 	{
 		double Norm = 0;
 		for(typename TVectorType::iterator iComponent = rThis.begin(); iComponent < rThis.end(); ++iComponent)
-		Norm += (*iComponent)*(*iComponent);
-		Norm = sqrt(Norm);
-		for(typename TVectorType::iterator iComponent = rThis.begin(); iComponent < rThis.end(); ++iComponent)
-		*iComponent /= Norm;
+			Norm += (*iComponent)*(*iComponent);
```

Comment:
Please reformat this as it used to be.

Reply:
do you mean the tab? I dont change any other than tabbing it for clarity of the loop. But sure, I can return it back.